basics/a26.go

```diff
@@ -0,0 +1,16 @@
+//This code uses some examples of displaying strings
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("/home" + "/foobar")
+	delimiter := "/"
```

Comment:
Let's use https://golang.org/pkg/path/filepath/ here. We would be exercising string anyway but with filepath it would look good.

Reply:
I don't have the module. Need to check how to install the module.